package.json

```diff
@@ -4,7 +4,8 @@
   "private": true,
   "scripts": {
     "start": "ts-node bin/www",
-    "build": "tsc -p ."
+    "build": "tsc -p .",
+    "test": "jest --passWithNoTests"
```

Comment:
What is the reason for the `--passWithNoTests` option?

Reply:
Removed --passWithNoTests option -
<img width="571" alt="image" src="https://user-images.githubusercontent.com/22402315/229726354-f562d97f-7cdf-4390-86f7-965874c41353.png">